CHANGELOG.md

```diff
@@ -1,5 +1,42 @@
 # Changelog
 
+# Version 1.0.0
+
+## Address
+
+Safe Locking Contract Address: [0x0a7CB434f96f65972D46A5c1A64a9654dC9959b2](https://etherscan.io/address/0x0a7CB434f96f65972D46A5c1A64a9654dC9959b2)
+
+## Compiler settings
+
+Same as before.
+
+## Constructor Argument
+
+1. Name: `initialOwner`
+   Type: `address`
+   Value: `0x8CF60B289f8d31F737049B590b5E4285Ff0Bd1D1`
+
+2. Name: `safeToken`
+   Type: `address`
+   Value: `0x5aFE3855358E112B5647B952709E6165e1c1eEEe`
+
+3. Name: `cooldownPeriod`
+   Type: `uint32`
+   Value: `86400`
+
+Deployment Parameters can also be found in [deployments.ts](./src/utils/deployment.ts)
```

Comment:
I would leave this out (or if you feel strongly about keeping it in, explain that we used the default parameters) - the linked file loads parameters from the environment and sets the defaults for unset parameters. Since we just used the default when deploying, the parameters can be found there, but I find this sentence a bit confusing in light of that.

Reply:
Removed 👍🏾